src/Symfony/Component/Messenger/DependencyInjection/MessengerPass.php

```diff
@@ -404,4 +414,9 @@ private function getServiceClass(ContainerBuilder $container, string $serviceId)
             return $definition->getClass();
         }
     }
+
+    private function isReceiverConsumable(array $tag): bool
+    {
+        return !isset($tag['is_consumable']) || false !== $tag['is_consumable'];
```

Comment:
Since you now have the `is_consumable`, this can be replaced with just `return $tag['is_consumable']`, making this method not really required, you can check the condition directly where the method is used.